src/accordion.ts

```diff
@@ -32,13 +32,13 @@ declare global {
 @customElement('glide-core-accordion')
 @final
 export default class Accordion extends LitElement {
-  /* c8 ignore start */
+  /* v8 ignore start */
```

Comment:
Migrated components can use the more standard `v8 ignore` instead of `c8 ignore`. 

We'll likely forget to make this change as we migrate components. So I also added a note to my post-migration todo list.